cli/src/cluster_query.rs

```diff
@@ -1179,13 +1183,26 @@ pub fn process_get_epoch_info(rpc_client: &RpcClient, config: &CliConfig) -> Pro
                 .get_recent_performance_samples(Some(60))
                 .ok()
                 .and_then(|samples| {
-                    let (slots, secs) = samples.iter().fold((0, 0), |(slots, secs), sample| {
-                        (slots + sample.num_slots, secs + sample.sample_period_secs)
-                    });
-                    (secs as u64).saturating_mul(1000).checked_div(slots)
```

Comment:
Looks like someone started the work here :sweat_smile:

Reply:
Sorry, I am not sure, I understand what do you mean.

Are you talking about the `sample: RpcPerfSample` deconstruction?
It seems to be simpler this way:

```rust
(
    slots.saturating_add(*num_slots),
    secs.saturating_add((*sample_period_secs).into()),
)
```

vs

```rust
(
    sample.slots.saturating_add(*sample.num_slots),
    sample.secs.saturating_add((*sample.sample_period_secs).into()),
)
```

I generally try to have a single function call or indirection per expression %)

Reply:
Oh sorry, I simply meant that it was funny to see one line of checked / saturating math in the middle of the whole file

Reply:
Ah, got it :)

Well, this is what the compiler told me to fix :)